src/coreclr/tools/superpmi/superpmi-shim-collector/icorjitinfo.cpp

```diff
@@ -1167,6 +1167,14 @@ void interceptor_ICJI::setVars(CORINFO_METHOD_HANDLE         ftn,   // [IN] meth
     original_ICorJitInfo->setVars(ftn, cVars, vars);
 }
 
+void interceptor_ICJI::reportRichMappings(ICorDebugInfo::InlineTreeNode*    inlineTreeNodes,
+                                          uint32_t                          numInlineTreeNodes,
+                                          ICorDebugInfo::RichOffsetMapping* mappings,
+                                          uint32_t                          numMappings)
+{
+    mc->cr->AddCall("reportRichMappings");
```

Comment:
Do you plan on eventually capturing this info? Seem fine to defer for now, but maybe add a TODO comment.

Reply:
Ah, good point, I forgot about this. Yes, let me add a todo and do it in a follow-up.
More importantly we need to free the arrays here.